_posts/2022-06-02-tf-2-9/tf-2-9.Rmd

```diff
@@ -0,0 +1,127 @@
+---
+title: "Tensorflow and Keras 2.9"
+description: > 
+  New tensorflow and keras releases bring improvements big and small.
+author:
+  - name: Tomasz Kalinowski
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com/
+slug: kalinowskiDLwR2e
+date: 2022-06-02
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+categories:
+  - Tensorflow
+  - Keras
+  - Deep Learning
+  - R
+editor_options:
+  markdown:
+    wrap: 72
+preview: images/coffee-n-milk.jpg
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, eval = FALSE)
+```
+
+```{r, echo = FALSE}
+library(tensorflow)
+library(keras)
+```
+
+## Tensorflow + Keras 2.9
+
+The release of the *Deep Learning with R, 2nd Edition* book coincides
+with new releases of Tensorflow and Keras. These releases bring many
+refinements that allow for more idiomatic and concise R code.
+
+The set of Tensor methods for base R generics has greatly expanded. The
```

Comment:
I'd use lower case and `code` formatting here, but I guess that's a matter of personal preference